test-fixture.html

```diff
@@ -235,11 +235,18 @@
 
     forcePolyfillAttachedStateSynchrony: function () {
       // Force synchrony in attachedCallback and detachedCallback where
-      // implemented, in the event that we are dealing with the async Web
-      // Components Polyfill.
-      if (window.CustomElements && window.CustomElements.takeRecords) {
+      // implemented, in the event that we are dealing with one of these async
+      // polyfills:
+      // 1. Web Components CustomElements polyfill (v1 or v0).
+      if (window.customElements && window.customElements.flush) {
+        window.customElements.flush();
```

Comment:
..this is not strictly needed, I was just getting ready for when this TODO will be done https://github.com/webcomponents/custom-elements/blob/master/src/custom-elements.js#L793